.github/workflows/on_discussion_create.yml

```diff
@@ -0,0 +1,40 @@
+name: Discussion auto-responder
+
+on:
+  discussion:
+    types: [created]
+
+env:
+  GITHUB_TOKEN: ${{ github.token }}
+
+jobs:
+  on_discussion_open:
+    runs-on: ubuntu-latest
+    permissions:
+      discussions: write
```

Comment:
Note: needed to be able to write discussion comments

Reply:
If we're using the service account to post as, we probably don't need this permissions line.